UnityProject/Assets/Scripts/Managers/MatrixManager/MatrixManager.cs

```diff
@@ -494,6 +494,7 @@ public static CustomPhysicsHit RayCast(Vector3 Worldorigin,
 			for (int i = 0; i < Count; i++)
 			{
 				var Info = loc[i];
+				if (Info.Matrix.IsSpaceMatrix) continue;
```

Comment:
should probably doc that this wont work in space

Reply:
there shouldn't be anything in space

Reply:
yes, but like I said; write it as a note in the function's header doc.